src/dataflow/analysis/tests/analysis-test.ts

```diff
@@ -394,11 +394,6 @@ describe('Solver', () => {
   });
 });
 
-const entityString = Flags.useNewStorageStack ?
```

Comment:
maybe have a function getEntityString() so you can run in the test as well as avoid the duplication. Alternatively initialise it in beforeEach

Reply:
Updated, thanks